tests/lib/TestUtils.cpp

```diff
@@ -26,7 +26,7 @@
 
 namespace testing {
 
-void _AssertDataEquals(
+void _AssertArrayEquals(
     const CPPUNIT_NS::SourceLine &source_line,
```

Comment:
Not yet. I'm still trying to decide how to unit test all of this. I'm going to experiment with a third unit testing framework so this will probably all change soon anyway.